### ISSUE TITLE
CommonJS Error when importing js-beautify

### ISSUE THREAD

**Issue by ekurutepe** [resend/resend-node#303]
Using `"node": "^21.2.0"` with TypeScript. Getting the following error:
```
"message":"Named export 'html' not found. The requested module 'js-beautify' is a CommonJS module, which may not support all module.exports as named exports. CommonJS modules can always be imported via the default export, for example using: import pkg from 'js-beautify'; const { html } = pkg; "
"stack":"file:///tmp/task/node_modules/@react-email/render/dist/index.mjs:49 import { html } from "js-beautify"; ^^^^ SyntaxError: Named export 'html' not found. The requested module 'js-beautify' is a CommonJS module, which may not support all module.exports as named exports. CommonJS modules can always be imported via the default export, for example using: import pkg from 'js-beautify'; const { html } = pkg; at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21) at ModuleJob.run (node:internal/modules/esm/module_job:190:5)"
"name":"SyntaxError"
```

My `tsconfig.json` looks like this: 
```
{
  "compilerOptions": {
    "target": "es6",
    "module": "commonjs",
    "outDir": "dist",
    "strict": true,
    "esModuleInterop": true
  },
  "include": ["**/*.ts"],
  "exclude": ["node_modules"]
}
```

Am I doing something wrong?

**Comment by ramnivas** [comment]
Facing the same issue (this one with Deno):
```
> import { Resend } from "npm:resend";
Uncaught SyntaxError: The requested module 'js-beautify' does not provide an export named 'html' at file:///Users/ramnivas/Library/Caches/deno/npm/registry.npmjs.org/@react-email/render/0.0.11/dist/index.mjs:49:10
    at async <anonymous>:1:41
```

**Comment by cpojer** [comment]
Same issue here.

Would it be possible to dump js-beautify or make it optional (via optional dep and try-catch a dynamic import)? It's a huuuge dependency and not needed for most use cases of resend.

**Comment by felix-dolderer** [comment]
> cpojer

This is especially true for devs not using react and thus not using the emails.create / batch.create functions which depend on @react-email/render which depends on js-beautify.

I don't even use the dependency for which the sub-dependency is broken.

Would be awesome to have a flag to instantiate Resend without the create functions and thus without the @react-email/render dependency.

**Comment by ITD-SKP-Project** [comment]
Facing the same issue when trying to import resend in SvelteKit. Is there a previous version of resend that doesn't have this error?

**Comment by felix-dolderer** [comment]
> Facing the same issue when trying to import resend in SvelteKit. Is there a previous version of resend that doesn't have this error? 

install `resend@2` there they use a different dependency

**Comment by gabrielmfern** [comment]
This is happening because of an import `@react-email/render` had and that is fixed on the latest, the Resend SDK just needs to be updated to use the latest as well and this should go away.
Maybe overriding the installed version of `@react-email/render` can be a workaround for now.

**Comment by JustMrMendez** [comment]
having the same problem on `"resend": "^3.0.0",`

**Comment by socratesomiliadis** [comment]
If anyone wants a temporary fix you can go to your package.json file and add this bit for npm:
```
 "overrides": {
      "@react-email/render": "0.0.12"
    }
```

or if you're using pnpm:

```
"pnpm": {
    "overrides": {
      "@react-email/render": "0.0.12"
    }
  }
```

**Comment by vcapretz** [comment]
hey there, we just released v3.1.0 with the updated `@react-email/render` version.
I'll close the issue now 😄
